### ISSUE TITLE
[Research/Audit] Tabbed Nav – Keybindings [II]

### ISSUE THREAD

**Issue by ssabo1** [sparkdesignsystem/spark-design-system#3586]
**User Story:**
Audit Spark Tabbed Navigation so that we can see how it is currently functioning with keybindings. 

**Notes:**
this is what should be: https://www.w3.org/TR/wai-aria-practices-1.1/#keyboard-interaction-19
Follow up with new issue.

**AC:**
Self audit Sparks tabbed nav component.

HTML:
How do Keybindings work today?
- Review what Spark has today.
- List what needs to be updated.

ANGULAR:
How do Keybindings work today?
- Review what Spark has today.
- List what needs to be updated.

REACT:
How do Keybindings work today?
- Review what Spark has today.
- List what needs to be updated.
